source/tyr/tests/integration/instance_test.py

```diff
@@ -193,6 +194,7 @@ def test_update_instances(create_instance):
     assert resp['max_car_no_park_direct_path_distance'] == 16000
     assert resp['places_proximity_radius'] == 10000
     assert resp['transfer_path'] is True
+    assert resp['access_points'] is True
```

Comment:
Minor: Another test with     assert resp['access_points'] is False will be preferable